manager/qemu/config.go

```diff
@@ -75,6 +82,7 @@ type Config struct {
 	UseSudo      bool   `env:"USE_SUDO"       envDefault:"false"`
 	EnableSEV    bool   `env:"ENABLE_SEV"     envDefault:"false"`
 	EnableSEVSNP bool   `env:"ENABLE_SEV_SNP" envDefault:"true"`
+	EnableTDX    bool   `env:"ENABLE_TDX"     envDefault:"false"`
```

Comment:
rather than bool values we can use manager to check for the device available where it is running `/dev/sev` or `/dev/tdx`. Then manager can implicitly know which configuration to apply. This also enables avoiding the error senario where a user sets all to true